falcon_kit/mains/pr_ctg_track.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import print_function
+from falcon_kit.multiproc import Pool
+import falcon_kit.util.io as io
+import argparse
+import sys
+import glob
+import os
+from heapq import heappush, heappop, heappushpop
+
+Reader = io.CapturedProcessReaderContext
+
+
+def get_pid_to_ctg(fn):
+    pid_to_ctg = {}
+    with open(fn) as f:
+        for row in f:
+            row = row.strip().split()
+            pid, rid, oid, ctg = row
+            pid_to_ctg.setdefault( pid, set()  )
+            pid_to_ctg[ pid ].add(ctg)
+    return pid_to_ctg
+
+def run_tr_stage1(db_fn, fn, min_len, bestn, pid_to_ctg):
+    cmd = "LA4Falcon -mo %s %s" % (db_fn, fn)
```

Comment:
You're right. M4OVLP (-m) is subtly different from FALCON (-f). And OVERLAP (-o) has a real effect too.